docs/modules/reference/pages/performance-data-collection/collectors/SnmpCollector.adoc

```diff
@@ -89,6 +89,30 @@ Ignored when using Newts as time-series storage.
 .Configuration overview for SNMP data collection
 image::performance-management/collectors/01_snmp-datacollection-configuration.png[]
 
+=== Support for opaque data types
+
+OpenNMS supports opaque data types including MIB import and data collection.
+Opauqe is ASN.1(Abstract Syntax Notation One) binary encoded data which is additionally wrapped in a hex string.
+The extracted from out of a hex string data can be of various types. 
+At the moment is the support for float data type implemented, but it can be easily extended if necessary. 
```

Comment:
```suggestion
OpenNMS currently supports the float data type, but this can be extended, if necessary. 
```
Does the user extend it, or is this something we do? 
If the user can extend it, we should provide an example of how. 
If we are planning to extend it in the future, then we should say:

"{page-component-title} currently supports the float data type; additional opaque data types will be supported in the future."